static/app/views/explore/metrics/metricRow.tsx

```diff
@@ -0,0 +1,72 @@
+import {useMemo} from 'react';
+
+import {SearchQueryBuilderProvider} from 'sentry/components/searchQueryBuilder/context';
+import {
+  TraceItemSearchQueryBuilder,
+  useSearchQueryBuilderProps,
+  type TraceItemSearchQueryBuilderProps,
+} from 'sentry/views/explore/components/traceItemSearchQueryBuilder';
+import {useMetricVisualize} from 'sentry/views/explore/metrics/metricsQueryParams';
+import {type TraceMetric} from 'sentry/views/explore/metrics/traceMetric';
+import {
+  useQueryParamsGroupBys,
+  useQueryParamsQuery,
+  useSetQueryParamsQuery,
+} from 'sentry/views/explore/queryParams/context';
+import {TraceItemDataset} from 'sentry/views/explore/types';
+
+interface MetricRowProps {
+  traceMetric: TraceMetric;
+}
+
+export function MetricRow({traceMetric}: MetricRowProps) {
+  const query = useQueryParamsQuery();
+  const setQuery = useSetQueryParamsQuery();
+
+  const tracesItemSearchQueryBuilderProps: TraceItemSearchQueryBuilderProps =
+    useMemo(() => {
+      return {
+        itemType: TraceItemDataset.TRACEMETRICS,
+        numberAttributes: {},
+        stringAttributes: {},
+        numberSecondaryAliases: {},
+        stringSecondaryAliases: {},
+        initialQuery: query,
+        onSearch: setQuery,
+        searchSource: 'ourmetrics',
```

Comment:
Kevan caught this on my stub PR, this should be 
```suggestion
        searchSource: 'tracemetrics',
```

but it's also not a big deal right now if you don't want to update this in this PR